kiwi/xml_description.py

```diff
@@ -105,11 +106,13 @@ def __parse(self):
                 '%s: %s' % (type(e).__name__, format(e))
             )
 
-    def __xsltproc(self):
-        Command.run(
-            [
-                'xsltproc', '-o', self.description_xslt_processed.name,
-                Defaults.get_xsl_stylesheet_file(),
-                self.description
-            ]
-        )
+    def _xsltproc(self):
+        '''
```

Comment:
Actually, having developing Salt, we hate double-quotes there and using single all everywhere, although PEP8 leans to the double-quotes. Old habits die hard...

While I fixed this, I would suggest not to make it religious, unless @schaefi has a strict PyLint on this.